src/dss/main.py

```diff
@@ -155,6 +156,35 @@ def stop_notebook_command(kubeconfig: str, notebook_name: str):
         exit(1)
 
 
+# FIXME: remove the `--kubeconfig`` option
+# after fixing https://github.com/canonical/data-science-stack/issues/37
+@main.command(name="start")
+@click.argument(
+    "name",
+    required=True,
+)
+@click.option(
+    "--kubeconfig",
+    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+)
+def start_notebook_command(name: str, kubeconfig: str):
+    """
+    Starts a stopped notebook in the DSS environment.
+    \b
+    Example:
+        dss start my-notebook
+    """
```

Comment:
I'm pretty sure you are following a standard, but printing the help for this yields the following:

```
$ dss start --help
Usage: dss start [OPTIONS] NAME

  Starts a stopped notebook in the DSS environment. Example:     dss start
  my-notebook

Options:
  --kubeconfig TEXT  Path to a Kubernetes config file. Defaults to the value
                     of the KUBECONFIG environment variable, else to
                     './kubeconfig'.
  --help             Show this message and exit.
```

Should we ensure the formatting is okay?

Reply:
fixed it

Reply:
```
dss stop --help
Usage: dss stop [OPTIONS] NOTEBOOK_NAME

  Stops a running notebook in the DSS environment.

  Example:
      dss stop my-notebook

Options:
  --kubeconfig TEXT  Path to a Kubernetes config file. Defaults to the value
                     of the KUBECONFIG environment variable, else to
                     './kubeconfig'.
  --help             Show this message and exit.
```